### PR TITLE
Map info

### PR DESCRIPTION
**Pull request by braitz**
This PR adds more provider map points, and displays the map-info sliding panel when map markers clicked.

I needed to go back to the static json loading to complete it.  I can re-integrate the parse after our styling deadlines, as needed.

cc @Chezlui @xnxky

### CODE REVIEW

File: app/src/main/java/org/ahomewithin/ahomewithin/models/User.java [hunk 1]
```diff
@@ -88,33 +104,119 @@ public static User getNewInstanceFromParseObject(ParseObjectUser poUser) {
         double lon = location == null ? 0 : location.getLongitude();
         double lan = location == null ? 0 : location.getLatitude();
         User newUser = new User(
-                poUser.getName(), poUser.getEmail(), poUser.getDesp(), poUser.getType(),
+                poUser.getName(), poUser.getEmail(), poUser.getDesp(), UserType.SERVICE_PROVIDER,
```
Comment (Chezlui) [inline app/src/main/java/org/ahomewithin/ahomewithin/models/User.java]: sure you want to hard code all the users to SERVICE_PROVIDERS?

**Comment by Chezlui** [comment]
I left a comment, the rest LGTM

File: app/src/main/java/org/ahomewithin/ahomewithin/util/MapMarkers.java [hunk 2]
```diff
@@ -121,19 +157,32 @@ public void onFailure(String error) {
...
+    private void restoreMarker(Marker marker, User user) {
+        if (marker != null) {
+//        switch(user.type) {
+//            case SERVICE_PROVIDER:
+//                markerOptions.icon(BitmapDescriptorFactory.defaultMarker(BitmapDescriptorFactory.HUE_GREEN));
+//                break;
+//            default: // COMMUNITY:
+//                markerOptions.icon(BitmapDescriptorFactory.defaultMarker(BitmapDescriptorFactory.HUE_ROSE));
+//        }
+            marker.setIcon(BitmapDescriptorFactory.defaultMarker(BitmapDescriptorFactory.HUE_GREEN));
```
Comment (Chezlui) [inline app/src/main/java/org/ahomewithin/ahomewithin/util/MapMarkers.java]: In this version you don't distinguish between users

**Comment by braitz** [comment]
Right now, all the users in the JSON file are providers.  This is OK.  We can extend it pretty easily again to add more "parent" community users.

My real goal was getting the map popup to work.

**Pull request merged**